File: Helper/SchoolService/SchoolAndroid/app/src/main/java/com/school/android/models/network/output/LoginModel.java
package com.school.android.models.network.output;

public class LoginModel {
    String login;
    String password;

    public String getLogin() {
        return login;
    }

    public void setLogin(String login) {
        this.login = login;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public LoginModel(String login, String password) {
        this.login = login;
        this.password = password;
    }
}

File: Helper/TranslateChatter/AuthAPI/Models/Controller/LoginModel.cs
using System.ComponentModel.DataAnnotations;

namespace AuthAPI.Models.Controller
{
    /// <summary>
    /// Логин
    /// </summary>
    public class LoginModel
    {
        [Required]
        public string Login { get; set; }
        [Required]
        public string Password { get; set; }
    }
}
